clients/web/csp.js

```diff
@@ -8,7 +14,7 @@ export function generateCSP(isDev = false) {
     directives: {
       "default-src": [SELF],
       "frame-src": [SELF],
-      "script-src": isDev ? [SELF, UNSAFE_EVAL] : [SELF],
+      "script-src": [SELF, WASM_UNSAFE_EVAL],
```

Comment:
we need to bring back `UNSAVE_EVAL` on dev or it will break HMR

Reply:
done!